client/scss/layouts/_login.scss

```diff
@@ -93,36 +141,11 @@ $desktop-nice-padding: 100px;
     display: none;
   }
 
-  // Special full-width, one-off fields i.e a single text or textarea input
-  .full input {
-    border-radius: 0;
-    font-weight: 300;
-    border: 0;
-    padding: 1.5em $mobile-nice-padding 1.5em $mobile-nice-padding;
-    font-size: 1.6em;
-    line-height: 1.6em;
-  }
-
-  // Forgotten password link
   .help {
-    font-size: 1.3em;
-    margin-top: 0;
-    padding: 10px 0;
-
-    @include media-breakpoint-up(sm) {
-      position: absolute;
-      top: 139px;
-      inset-inline-end: 5%;
-      padding: 0;
-    }
-
-    &__link {
-      color: $color-white;
-
-      @include media-breakpoint-up(sm) {
-        color: $color-link;
-      }
-    }
+    padding-left: 4.3em;
```

Comment:
Avoid padding left. Instead use padding start or just padding. This needs to work for RTL users also.